### PR TITLE
Image uploader picker component for avatar

### PR DESCRIPTION
**Pull request by carlosascari**
The ImagePicker is usable however it still requires a server side endpoint to be developed for social media sources, for now it is mocked so you can interact with it.

### CODE REVIEW

**Comment by asood123** [comment]
Is this ready to be reviewed and merged? The backend has been merged already.

**Comment by carlosascari** [comment]
@asood123 Yes it is ready to be reviewed and merged

File: frontend/src/components/public_group/PublicGroupSignupV2.js [hunk 1]
```diff
@@ -20,53 +21,102 @@ export default class PublicGroupSignupV2 extends React.Component {
...
+                  handleChange={twitterHandle => appendProfileForm({twitterHandle})}/>
+              </div>              
+            </div>
+
           </div>
           <div className="buttonsRow pt3">
             <SaveButton
               save={save.bind(this)}
-              label={i18n.getString('save')}
+              label="SHOW MY SUPPORT!" //  {/*i18n.getString('save')*/}
```
Comment (xdamman) [inline frontend/src/components/public_group/PublicGroupSignupV2.js]: Why?

Comment (carlosascari) [inline frontend/src/components/public_group/PublicGroupSignupV2.js]: My mistake, I changed the text to match the text in the design that cuiki gave me, I forgot to add it to the `strings.json` file. I'll add it now, unless you you want to keep the original string `Save` ?

Comment (xdamman) [inline frontend/src/components/public_group/PublicGroupSignupV2.js]: Yes keep "save"

Comment (carlosascari) [inline frontend/src/components/public_group/PublicGroupSignupV2.js]: "Save" string has been restored

**Pull request merged**